lib/controllers/profiles.ts

```diff
@@ -59,7 +59,9 @@ import { ServiceModel } from "../models/service";
 function toExtendedProfile(profile: RetrievedProfile): ExtendedProfile {
   return {
     email: profile.email,
-    version: profile.version
+    version: profile.version,
+    is_storage_of_message_content_enabled:
+      profile.isStorageOfMessageContentEnabled
```

Comment:
again I would give it a feature name, like `inbox` -> `isInboxEnabled`

"Inbox" may not be only about storing messages, the feature may provide other functionalities (e.g. search?), so it makes sense to use a name that is decoupled from a specific functionality